src/config.ts

```diff
@@ -12,6 +12,8 @@ const notifications = {
 	email: email.username && email.password
 };
 
+const navigationTimeout = 60000;
```

Comment:
Let's put this inside the `page` variable

Reply:
done